samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobSample.java

```diff
@@ -0,0 +1,122 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package aiplatform;
+
+// [START aiplatform_create_batch_prediction_job_sample]
+import com.google.cloud.aiplatform.v1beta1.AcceleratorType;
+import com.google.cloud.aiplatform.v1beta1.BatchDedicatedResources;
+import com.google.cloud.aiplatform.v1beta1.BatchPredictionJob;
+import com.google.cloud.aiplatform.v1beta1.GcsDestination;
+import com.google.cloud.aiplatform.v1beta1.GcsSource;
+import com.google.cloud.aiplatform.v1beta1.JobServiceClient;
+import com.google.cloud.aiplatform.v1beta1.JobServiceSettings;
+import com.google.cloud.aiplatform.v1beta1.LocationName;
+import com.google.cloud.aiplatform.v1beta1.MachineSpec;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+import java.io.IOException;
+import javax.json.Json;
+import javax.json.JsonObject;
+
+public class CreateBatchPredictionJobSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "PROJECT";
+    String displayName = "DISPLAY_NAME";
+    String modelName = "MODEL_NAME";
+    String instancesFormat = "INSTANCES_FORMAT";
+    String gcsSourceUri = "GCS_SOURCE_URI";
+    String predictionsFormat = "PREDICTIONS_FORMAT";
+    String gcsDestinationOutputUriPrefix = "GCS_DESTINATION_OUTPUT_URI_PREFIX";
+    createBatchPredictionJobSample(
+        project,
+        displayName,
+        modelName,
+        instancesFormat,
+        gcsSourceUri,
+        predictionsFormat,
+        gcsDestinationOutputUriPrefix);
+  }
+
+  static void createBatchPredictionJobSample(
+      String project,
+      String displayName,
+      String modelName,
+      String instancesFormat,
+      String gcsSourceUri,
+      String predictionsFormat,
+      String gcsDestinationOutputUriPrefix)
+      throws IOException {
+    JobServiceSettings settings =
+        JobServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+    String location = "us-central1";
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (JobServiceClient client = JobServiceClient.create(settings)) {
+      JsonObject jsonModelParameters = Json.createObjectBuilder().build();
+      Value.Builder modelParametersBuilder = Value.newBuilder();
+      JsonFormat.parser().merge(jsonModelParameters.toString(), modelParametersBuilder);
```

Comment:
This is pretty confusing - it looks like jsonModelParameters would be something empty?

Reply:
Yeah, that is correct. You need to pass an empty Value object to the service for that field.

I'll add a note that clarifies this.

Reply:
Done.